packages/cysync-core/src/constants/routes/index.ts

```diff
@@ -3,7 +3,7 @@ import { OnboardingRouteName, onboarding } from './onboarding';
 const rootRoutes = {
   permissionSetup: {
     name: 'permission-setup',
-    path: '/',
+    path: '/permission-setup',
```

Comment:
This should be the default route

Reply:
But, then it is causing error in `e2e` test.